src/rule-helpers.js

```diff
@@ -20,7 +20,15 @@ function takeWhile(source, predicate, excludes) {
 
 // Retrieves style guide documentation with cached responses
 export default async function getRuleMarkDown(url) {
-  const rule = url.split('#')[1]
+  if (url == null) {
+    return null
+  }
+  const ruleMatch = /https:\/\/github.com\/.*\/ruby-style-guide#(.*)/g.exec(url)
```

Comment:
This match is doing the same thing as what the previous `split` did... as long as the rule name doesn't contain `#` as well (which would be invalid coming from Rubocop anyway). Is there a reason to make this change?

The `null` checks are great though.

Reply:
I did the change to use regex because the URL could be an external to rubocop style guide (from Wikipedia or another) and could contain a sharp too, in that case no rule markdown will be available and the function result will be null.